assets/theme-supplement.css

```diff
@@ -529,4 +539,43 @@
   }
 }
 
+.reviews-title {
```

Comment:
This class is not following the BEM convention. Where is being used btw?

Reply:
I forgot to remove this one.